jetstream/core/orchestrator.py

```diff
@@ -147,6 +148,8 @@ class ActiveRequest:
   """Current state of the driver."""
 
   #################### Information relevant for generation #####################
+  # The unique id for the activeRequest
```

Comment:
Add some comments on how it is used and why it is needed

Reply:
The code below indicates this can be None. If so, have the right annotation

Reply:
Sure, will add more comments. The request id itself will not be none, but we could pass none into the engine api